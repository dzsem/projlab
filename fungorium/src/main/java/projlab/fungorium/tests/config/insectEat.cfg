# Insect Eat
# ------------------------------------------
# Init state
# ------------------------------------------
# 0
ADD TECTON
# 1
ADD INSECT 0 0
# 2
ADD MUSHROOMSPORE 0
SET EFFECTGENERATION 2 NONE