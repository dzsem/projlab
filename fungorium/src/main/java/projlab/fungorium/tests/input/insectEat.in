# Insect Eat
# ------------------------------------------
# Input
# ------------------------------------------
LOAD config/insectEat.cfg
INSECT 1 EATSPORE
# Print states
# ------------------------------------------
SAVE insectEat.out