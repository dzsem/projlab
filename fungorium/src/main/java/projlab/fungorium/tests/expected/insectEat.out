TECTON 0 0 -1 0 0
INSECT 1 0 0 1 1 0